### ISSUE TITLE
Add to F-Droid

### ISSUE THREAD

**Issue by yarons** [patzly/grocy-android#95]
An Open Source Android Apps Store:
https://f-droid.org/

This is how it's done:
https://f-droid.org/en/docs/FAQ_-_App_Developers/#how-do-i-get-my-app-included

**Comment by yarons** [comment]
Sorry, I just saw your comment about it on the Readme.

**Comment by patzly** [comment]
Thanks for the hint! We have read a lot about it, but it takes time and is a bit complicated, so it is not our top priority.

**Comment by timokau** [comment]
Is there some upstream issue or similar that I can follow to get a notification once it lands on f-droid?

**Comment by MrJohnZoidberg** [comment]
Not yet, but I can reopen this issue and we will notify you here when it's on F-Droid.

**Comment by timokau** [comment]
Thank you!

**Comment by MrJohnZoidberg** [comment]
Hello @IzzySoft,
I'd like to include our app in the official F-Droid repo, so I've created the fastlane metadata structure.
Now I wonder, what I should do next:
- Release a new version of the app (to create a new tag with the metadata)
- Create a new issue [here](https://gitlab.com/fdroid/rfp/-/issues)

If this is right, do I have to fill the fields `Name`, `Summary` or `Description` in the issue? Because they are already in the fastlane metadata (and translated)...

Thanks for your help!

**Comment by IzzySoft** [comment]
Just create the issue. No need to fill summary/description if at the point of opening the RFP Fastlane is already available and tagged here. Let the bot take care to check for compliance, and me for setting up metadata then.

**Comment by IzzySoft** [comment]
PS: You forgot `changelogs/21.txt` when making today's release :wink:

**Comment by MrJohnZoidberg** [comment]
@IzzySoft  Oh sorry, I've done that now and recreated the tag. Thanks for reminding me!

**Comment by IzzySoft** [comment]
My updater always informs me about missing changelogs. I've already manually added one here, based on what the tag reported. That won't happen over at F-Droid.org then, btw :wink:

**Comment by MrJohnZoidberg** [comment]
Great news: Our app is now in F-Droid, thanks to the great F-Droid team!

It may take some days until you can download it through the F-Droid store.

**Comment by IzzySoft** [comment]
Let's hope our current "traditional" called "could not bring up…" doesn't dealy the build too much…

**Comment by timokau** [comment]
It's available now! Thank you @MrJohnZoidberg and @IzzySoft :rocket:

**Comment by IzzySoft** [comment]
Congrats :congratulations: :grinning:

@MrJohnZoidberg At this point I usually remove apps from my repo (after giving them a few days to settle, and the users to notice) – unless there's demand to keep it (e.g. to cater stable versions via F-Droid, while having "early birds" for testers in my repo). Please let me know what you'd prefer :wink:

**Comment by MrJohnZoidberg** [comment]
I think you can remove our app from your repository. Thanks a lot for your help and service!:rocket:

One last question:
I have seen with the app "Zapp" in F-Droid that it is possible to release a alpha or beta version so that only users with the pre-release option enabled (in the F-Droid app) will get these versions. But how can I accomplish this?

**Comment by IzzySoft** [comment]
> Thanks, I think you can remove our app from your repository.

Done – effective with tomorrow's sync. Should you later need it back, simply yell: I've kept metadata, so it should be an easy thing.

> Thank you very much for your help and service!

Gladly!

> But how can I accomplish this?

Wild guess: manual MR adding the build without increasing CurrentVersion/CurrentVersionCode at the end of the YAML. Actually, not that wild a guess, but rather what I remember having seen. That way, CV/CVC (aka "suggested version" in the client) still point to the latest stable, while the newer alpha/beta is available for manual updates. That way I usually run the latest alpha of the F-Droid client.
